### ISSUE TITLE
Wrong number of total steps for linear warmup schedule 

### ISSUE THREAD

**Issue by tholor** [deepset-ai/FARM#46]
The number of total steps is currently wrongly calculated due to wrong rounding here: https://github.com/deepset-ai/FARM/blob/90ecbeba0785aced15c27ba5710068844ace8c09/farm/experiment.py#L256

It's resulting in one step less than required per epoch. If running for multiple epochs, this will result in LR = 0 for the last few steps. The related warning message occurs:
 `Training beyond specified 't_total'. Learning rate multiplier set to 0. Please set 't_total' of {} correctly.`

In addition, we start with a LR of zero for the first step which is a waste of computation.
![wrong_lr](https://user-images.githubusercontent.com/1563902/62549322-8f316780-b868-11e9-885a-86f87e164ca7.png)

**Comment by tholor** [comment]
Fixed by #54
